sqlwhat/check_result.py

```diff
@@ -0,0 +1,67 @@
+from pythonwhat.Test import TestFail, Test
+
+def check_result(state, msg="Incorrect result."):
+    stu_res = state.student_result
+    sol_res = state.solution_result
+
+    # empty test
+    test_has_columns(state)
```

Comment:
Good question! My sense is that `check_result` should allow customization, in the form of changing the arguments to subtests--with the goal that 

* `check_result` is the function that works most of the time
* the source code of `check_result` is very clearly just passing arguments to subtests

I could definitely see it only containing all message customization. Seems simple but what people will want most the time.